src/main/java/com/poortorich/global/date/constants/DateConstants.java

```diff
@@ -1,8 +1,17 @@
 package com.poortorich.global.date.constants;
 
+import java.time.Month;
+import java.util.List;
+
 public class DateConstants {
 
     public static final int FIRST_DAY_OF_YEAR = 1;
     public static final int FIRST_DAY_OF_MONTH = 1;
     public static final int ONE_DAY = 1;
+
+    public static final List<Month> MONTHS_ORDERED = List.of(
+            Month.JANUARY, Month.FEBRUARY, Month.MARCH, Month.APRIL,
+            Month.MAY, Month.JUNE, Month.JULY, Month.AUGUST,
+            Month.SEPTEMBER, Month.OCTOBER, Month.NOVEMBER, Month.DECEMBER
+    );
```

Comment:
나중에 수정하시면 됩니다만 아래와 같이 수정하면 깔끔합니다.

```suggestion
    public static final List<Month> MONTHS_ORDERED = Arrays.asList(Month.values()));
```